packages/marshal/package.json

```diff
@@ -77,6 +77,6 @@
     "timeout": "2m"
   },
   "typeCoverage": {
-    "atLeast": 89.31
+    "atLeast": 85.3
```

Comment:
Here it was `86.91` just before the fix, so it's a real drop. I imagine because more types were discovered, so even though the numerator increased the denominator increased more.